lib/Event/AdminEvents.php

```diff
@@ -143,6 +143,34 @@ final class AdminEvents
      */
     const OBJECT_LIST_BEFORE_LIST_LOAD = 'pimcore.admin.object.list.beforeListLoad';
 
+    /**
+     * Allows you to modify the object list before it is prepared for export.
+     *
+     * Subject: A controller extending \Pimcore\Bundle\AdminBundle\Controller\AdminController
+     * Arguments:
+     *  - list | the object list
+     *  - context | contains contextual information
+     *
+     * @Event("Pimcore\Event\Model\GenericEvent")
```

Comment:
```suggestion
     * @Event("Symfony\Component\EventDispatcher\GenericEvent")
```

Reply:
Will change it, but the file contains much more Pimcore\Event\Model\GenericEvent references.

Reply:
I have created a PR to fix other referenceshttps://github.com/pimcore/pimcore/pull/8287/files#diff-c78edb95162e7fb08aa02612d5a6b4791cc83d6f4a5ed3392918c47fce39e929